tests/test_models/test_base_model.py

```diff
@@ -2,16 +2,18 @@
 """Test BaseModel for expected behavior and documentation"""
 from datetime import datetime
 import inspect
-import models
+from models.base_model import BaseModel
 import pep8 as pycodestyle
 import time
-import unittest
+import json
+import os
+from unittest import TestCase
 from unittest import mock
```

Comment:
This line and the one above should be merged into `from unittest import TestCase, mock`.

Reply:
Yes, great way to save an extra line of code!